automation/check-patch.e2e-ocp.sh

```diff
@@ -1,33 +0,0 @@
-#!/bin/bash -xe
```

Comment:
We indeed should drop this burden (and from infra as well). However, do it please in a different PR